From gadiyar@ti.com  Wed Feb 17 15:57:55 2010
From: Anand Gadiyar <gadiyar@ti.com>
Date: Fri, 12 Feb 2010 17:54:59 +0530
Subject: USB: omap: ehci: kill 2 compile warnings
Cc: Anand Gadiyar <gadiyar@ti.com>
Message-ID: <1265977499-10791-1-git-send-email-gadiyar@ti.com>


Kill these compile warnings:
CC [M]  drivers/usb/host/ehci-hcd.o
drivers/usb/host/ehci-dbg.c:45: warning: 'dbg_hcs_params' defined but not used
drivers/usb/host/ehci-dbg.c:89: warning: 'dbg_hcc_params' defined but not used

Signed-off-by: Anand Gadiyar <gadiyar@ti.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-omap.c |    3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/usb/host/ehci-omap.c
+++ b/drivers/usb/host/ehci-omap.c
@@ -645,6 +645,9 @@ static int ehci_hcd_omap_probe(struct pl
 	omap->ehci->regs = hcd->regs
 		+ HC_LENGTH(readl(&omap->ehci->caps->hc_capbase));
 
+	dbg_hcs_params(omap->ehci, "reset");
+	dbg_hcc_params(omap->ehci, "reset");
+
 	/* cache this readonly data; minimize chip reads */
 	omap->ehci->hcs_params = readl(&omap->ehci->caps->hcs_params);
 
